pkg/validation/internal/removed_apis.go

```diff
@@ -78,34 +87,22 @@ func validateDeprecatedAPIs(bundle *manifests.Bundle, k8sVersion string) errors.
 }
 
 // validateDeprecatedAPIS will check if the operator bundle is using a deprecated or no longer supported k8s api
-// Note if the k8s was informed via "k8s=1.22" it will be used. Otherwise, we will use the minKubeVersion in
+// Note if the k8s was informed via "k8s-version" optional key it will be used. Otherwise, we will use the minKubeVersion in
```

Comment:
```suggestion
// Note if the k8s version was informed via "k8s-version" optional key it will be used. Otherwise, we will use the minKubeVersion in
```